integrated_channels/sap_success_factors/exporters/learner_data.py

```diff
@@ -42,14 +43,29 @@ def get_learner_data_record(self, enterprise_enrollment, completed_date=None, gr
                 'sap_success_factors',
                 'SapSuccessFactorsLearnerDataTransmissionAudit'
             )
-            return SapSuccessFactorsLearnerDataTransmissionAudit(
-                enterprise_course_enrollment_id=enterprise_enrollment.id,
-                sapsf_user_id=sapsf_user_id,
-                course_id=enterprise_enrollment.course_id,
-                course_completed=course_completed,
-                completed_timestamp=completed_timestamp,
-                grade=grade,
-            )
+            # Since we have started sending courses to SuccessFactors instead of course runs,
```

Comment:
I like the amount of detail we've included in this comment, but it seems misplaced to me -- should it be moved to the transmitter, vs. the exporter?

Reply:
Also I'm probably just missing it, but I don't see the retry logic for when the first transmission fails?

Reply:
Yea, I guess this is ultimately depending on the logic in the transmitter, which is receiving an iterable of course transmissions to try.
In the case that the course key transmission succeeds, the course run id transmission will not get attempted because a transmission by that enrollment id has already succeeded. In the case that the course key transmission fails, there is no successful transmission for the enrollment id and so we try the course run id transmission.

Reply:
Maybe we can add some comments explaining this?